database/sql/V044__limit_order_execution.sql

```diff
@@ -0,0 +1,10 @@
+-- Like orders.surplus_fee but is the value used to execute a limit order in a specific auction.
+-- Frontend requested to have access to this.
+
+ALTER TABLE order_rewards
+RENAME TO order_execution
+;
```

Comment:
@bh2smith I think this is a (simple) breaking change for the rewards script. Is that ok? Context is that we need to store more per-order-per-auction stuff for which we want to use the same table but then the old name doesn't make sense anymore.

Reply:
Yes - this is definitely a breaking change. currently we execute Identical queries on both DBs, so also we would have to adapt this and parameterize the rewards table-name. This is going to be a non-trivial change from our side, but I will get started on it.

Reply:
It seems a lot like this change is going to be some temporary fix that we will revert once both tables have the same name. This is not really ideal, but it can be done.

Reply:
Fixed, not so difficult code wise (but deployments will have to be coordinated):

https://github.com/cowprotocol/solver-rewards/pull/140

Reply:
Does it have to be coordinated during the week or only for the next payout run?

Reply:
I think it needs to coordinate during the week because of the "missing entries" alert that we have.

Reply:
Actually it doesn't matter anymore I have modified the code to work with both versions of the table over both environments (via try querying new table name , otherwise query with old table name). This should be a seemless change from our side.